### PR TITLE
getCurrentLanguageCode() for all platforms return iso 639-1 language cod...

### PR DESCRIPTION
**Pull request by newnon**
...e

iPhone template improvements(for game center support)

### CODE REVIEW

File: cocos/2d/platform/win32/CCApplication.cpp [hunk 1]
```diff
@@ -190,6 +190,16 @@ LanguageType Application::getCurrentLanguage()
...
 
+const char * Application::getCurrentLanguageCode()
+{
+	LANGID lid = GetUserDefaultUILanguage();
+	const LCID locale_id = MAKELCID(lid, SORT_DEFAULT);
+	static char code[3] = { 0 };
+	GetLocaleInfoA(locale_id, LOCALE_SISO639LANGNAME, code, sizeof(code));
+	code[2] = '\0';
+	return code;
+}
```
Comment (dumganhar) [inline cocos/2d/platform/win32/CCApplication.cpp:201]: `cocos/2d/platform/win32/CCApplication.cpp  100644 → 100755` 
Please don't change the file mode. Thanks.

**Comment by newnon** [comment]
whats course i changed in on windows machine do i need make a new pull request?

**Comment by dumganhar** [comment]
No, just change it back, push to the same branch again.
BTW, do we need this method since we have getCurrentLanguage method yet?

**Comment by newnon** [comment]
many sdk have some methods that need iso 639-1 code and for this moment we need to write ugly code like this 
switch(getCurrentLanguage())
{
    case LanguageType::ENGLISH:
    return "en"; 
}
and also not all languages are in you enum
at this moment i finishing new SpriteBuilder reader and it also store his languages files with iso code

**Comment by dumganhar** [comment]
That makes sense.

**Comment by dumganhar** [comment]
[ci rebuild]

**Comment by newnon** [comment]
[ci rebuild] ?

**Comment by dumganhar** [comment]
ci rebuild is used for restarting our jenkin ci machine.

**Comment by dumganhar** [comment]
Thanks.

**Pull request merged**